code/datums/gamemodes/crash.dm

```diff
@@ -0,0 +1,289 @@
+
+
+/datum/game_mode/crash/declare_completion()
+	. = ..()
+	to_chat(world, "<span class='round_header'>|[round_finished]|</span>")
+	to_chat(world, "<span class='round_body'>Thus ends the story of the brave men and women of the TGS Cantebury and their struggle on [SSmapping.configs[GROUND_MAP].map_name].</span>")
+	
+	// Music
+	var/xeno_track
+	var/human_track
+	switch(round_finished)
+		if(MODE_CRASH_X_MAJOR)
+			xeno_track = pick('sound/theme/winning_triumph1.ogg','sound/theme/winning_triumph2.ogg')
+			human_track = pick('sound/theme/sad_loss1.ogg','sound/theme/sad_loss2.ogg')
+		if(MODE_CRASH_M_MAJOR)
+			xeno_track = pick('sound/theme/sad_loss1.ogg','sound/theme/sad_loss2.ogg')
+			human_track = pick('sound/theme/winning_triumph1.ogg','sound/theme/winning_triumph2.ogg')
+		if(MODE_CRASH_X_MINOR)
+			xeno_track = pick('sound/theme/neutral_hopeful1.ogg','sound/theme/neutral_hopeful2.ogg')
+			human_track = pick('sound/theme/neutral_melancholy1.ogg','sound/theme/neutral_melancholy2.ogg')
+		if(MODE_CRASH_M_MINOR)
+			xeno_track = pick('sound/theme/neutral_melancholy1.ogg','sound/theme/neutral_melancholy2.ogg')
+			human_track = pick('sound/theme/neutral_hopeful1.ogg','sound/theme/neutral_hopeful2.ogg')
+		if(MODE_CRASH_DRAW_DEATH)
+			xeno_track = pick('sound/theme/nuclear_detonation1.ogg','sound/theme/nuclear_detonation2.ogg') 
+			human_track = pick('sound/theme/nuclear_detonation1.ogg','sound/theme/nuclear_detonation2.ogg')
+
+	for(var/i in GLOB.xeno_mob_list)
+		var/mob/M = i
+		SEND_SOUND(M, xeno_track)
+
+	for(var/i in GLOB.human_mob_list)
+		var/mob/M = i
+		SEND_SOUND(M, human_track)
+
+	for(var/i in GLOB.observer_list)
+		var/mob/M = i
+		if(ishuman(M.mind.current))
+			SEND_SOUND(M, human_track)
+			continue
+
+		if(isxeno(M.mind.current))
+			SEND_SOUND(M, xeno_track)
+			continue
+
+		SEND_SOUND(M, pick('sound/misc/gone_to_plaid.ogg', 'sound/misc/good_is_dumb.ogg', 'sound/misc/hardon.ogg', 'sound/misc/surrounded_by_assholes.ogg', 'sound/misc/outstanding_marines.ogg', 'sound/misc/asses_kicked.ogg'))
+
+	
+	log_game("[round_finished]\nGame mode: [name]\nRound time: [duration2text()]\nEnd round player population: [length(GLOB.clients)]\nTotal xenos spawned: [GLOB.round_statistics.total_xenos_created]\nTotal humans spawned: [GLOB.round_statistics.total_humans_created]")
+
+	announce_medal_awards()
+	announce_round_stats()
+	end_of_round_deathmatch()
+
+
+/datum/game_mode/crash/proc/initialize_xeno_leader()
+	var/list/possible_queens = get_players_for_role(BE_QUEEN)
+	if(!length(possible_queens))
+		return FALSE
+
+	var/found = FALSE
+	for(var/i in possible_queens)
+		var/datum/mind/new_queen = i
+		if(new_queen.assigned_role || is_banned_from(new_queen.current?.ckey, ROLE_XENO_QUEEN))
+			continue
+		if(queen_age_check(new_queen.current?.client))
+			continue
+		new_queen.assigned_role = ROLE_XENO_QUEEN
+		xenomorphs += new_queen
+		found = TRUE
+		break
+
+	return found
+
+/datum/game_mode/crash/proc/initialize_xenomorphs()
```

Comment:
Move this to /game_mode and perhaps turn it into a bitflag if you want a gamemode to have xenos